app/uk/gov/hmrc/incorporatedentityidentificationfrontend/repositories/JourneyConfigRepository.scala

```diff
@@ -109,6 +113,26 @@ object JourneyConfigRepository {
     }
   }
 
-  implicit val journeyConfigMongoFormat: OFormat[JourneyConfig] = Json.format[JourneyConfig]
+  implicit val journeyConfigMongoFormat: OFormat[JourneyConfig] = new OFormat[JourneyConfig] {
+    override def reads(json: JsValue): JsResult[JourneyConfig] =
+      for {
+        continueUrl <- (json \ ContinueUrlKey).validate[String]
+        pageConfig <- (json \ PageConfigKey).validate[PageConfig]
+        businessEntity <- (json \ BusinessEntityKey).validate[BusinessEntity]
+        businessVerificationCheck <- (json \ BusinessVerificationCheckKey).validate[Boolean]
+        regime <-  (json \ RegimeKey).validateOpt[String]
+      } yield {
+        JourneyConfig(continueUrl, pageConfig, businessEntity, businessVerificationCheck, regime.getOrElse("VATC"))
```

Comment:
As regime is a required parameter I assume this is to protect user's who are on the journey when the code changes.

Reply:
Yeah. We can then remove this again once this has been successfully deployed. Just to not interrupt the users who are mid flow when we deploy it